src/config.ts

```diff
@@ -100,6 +100,10 @@ export const LIQUIDITY_POOL_REGISTRY_ADDRESS: string | undefined = _.isEmpty(
           EnvVarType.ETHAddressHex,
       );
 
+export const RFQT_API_KEY_WHITELIST: string[] = JSON.parse(process.env.RFQT_API_KEY_WHITELIST || '[]');
```

Comment:
Consider allowing simple csv in the env var instead of requiring full fledged JSON.

Reply:
CSV is what we typically use.

Reply:
Addressed in https://github.com/0xProject/0x-api/pull/162/commits/47227ad4e38d79b72d4ebe0629638433e6606a51